docs/api/hardhat/hardhat-zksync-upgradable.md

```diff
@@ -130,7 +129,7 @@ After that, load the `Box` artifact and call the `deployProxy` method from the `
 ```typescript
   const contractName = 'Box';
   const contract = await deployer.loadArtifact(contractName);
-  await hre.zkUpgrades.deployProxy(deployer.zkWallet, contract, [42], { initializer: 'store' });
+  await hre.zkUpgrades.deployProxy(deployer.zkWallet, contract, [42], { initializer: 'initialize' });
```

Comment:
is this an API change from our side?

Reply:
ohh I didn't realised this was the method from the contract. Thanks for the info